submarine-workbench/workbench-web/src/app/services/user.service.ts

```diff
@@ -96,6 +96,7 @@ export class UserService {
 
   createUser(sysUser: Partial<SysUser>): Observable<SysUser> {
     const apiUrl = this.baseApi.getRestApi('/sys/user/add');
+    sysUser['password'] = md5(sysUser.password);
```

Comment:
The user password stored in the database is already encrypted.

Reply:
It seems like the password is not encrypted while creating a new user.
![password](https://user-images.githubusercontent.com/50860453/97256954-691e2b80-184f-11eb-9803-72809330de15.png)